animate-config.json

```diff
@@ -3,8 +3,10 @@
     "bounce": true,
     "flash": true,
     "pulse": true,
+    "press": true,
```

Comment:
This file is no more on the dev branch. Make sure you update https://github.com/daneden/animate.css/blob/dev/source/animate.css instead.